policies_examples.md

```diff
@@ -0,0 +1,105 @@
+
+```
+<scenario>
+   <world>
+      <region name="USA">
+         <linked-ghg-policy name="CO2">
+            <price-adjust fillout="1" year="1975">1</price-adjust>
+            <demand-adjust fillout="1" year="1975">3.667</demand-adjust>
+            <market>global</market>
+            <linked-policy>GHG</linked-policy>
+            <price-unit>1990$/tC</price-unit>
+            <output-unit>MtC</output-unit>
+         </linked-ghg-policy>
+         <linked-ghg-policy name="CH4">
+            <price-adjust fillout="1" year="1975">5.7272</price-adjust>
+            <demand-adjust fillout="1" year="1975">21</demand-adjust>
+            <market>global</market>
+            <linked-policy>GHG</linked-policy>
+            <price-unit>1990$/GgCH4</price-unit>
+	...
+```
+
+## <a name="energy-constraint"> Energy Constraint </a>
+The following inputs will set a constraint on bioenergy use in the USA, limiting it to 10 EJ/yr. Note that the `input-tax` tag will need to be added to all model periods (the example only uses 2020 for brevity). Additionally, this tag needs to be added to all production technologies and regions that are included in the target. For example, if you wanted to include 1st generation bioenergy in this constraint, you would need to add a tag to those technologies in the refinery sector, as 1st generation bioenergy does not go through the "regional biomass" market. If you wanted to only constrain one type of bioenergy, then would only put the `input-tax` tag in the technology producing that type of bioenergy (e.g., you could include a technology with this `input-tax` in the biomass resource to limit MSW production only). This example sets an upper bound on production. If instead you wanted a lower bound, then you would use `input-subsidy` in the technology and `policyType` equals "subsidy" in the policy-portfolio-standard. 
+
+```
+<scenario>
+   <world>
+      <region name="USA">
+         <supplysector name="regional biomass">
+            <subsector name="regional biomass">
+               <technology name="regional biomass">
+                  <period year="2020">
+                     <input-tax name="bio-constraint">
+                        <coefficient>1.00</coefficient>
+                     </input-tax>
+                  </period>
+                  ...
+               </technology>
+            </subsector>
+        </supplysector>
+        <policy-portfolio-standard name="bio-constraint">
```

Comment:
Interesting, this is not at all how I do it?  Do both methods do the same thing @pralitp ?

Here is how I set a floor on corn ethanol  (which is also a ceiling as it has a negative minimum price)

       <policy-portfolio-standard name="Corn-Ethanol-Floor">
            <policyType>subsidy</policyType>
            <market>USA</market>
            <min-price year="2015" fillout="1">-100</min-price>
            <constraint year="2015">1.323</constraint>
            <constraint year="2020">1.524</constraint>
            <constraint year="2025">1.524</constraint>
            <constraint year="2030">1.524</constraint>
            <constraint year="2035">1.524</constraint>
            <constraint year="2040">1.524</constraint>
            <constraint year="2045">1.524</constraint>
            <constraint year="2050">1.524</constraint>
            <constraint year="2055">1.524</constraint>
            <constraint year="2060">1.524</constraint>
            <constraint year="2065">1.524</constraint>
            <constraint year="2070">1.524</constraint>
            <constraint year="2075">1.524</constraint>
            <constraint year="2080">1.524</constraint>
            <constraint year="2085">1.524</constraint>
            <constraint year="2090">1.524</constraint>
            <constraint year="2095">1.524</constraint>
            <constraint year="2100">1.524</constraint>
         </policy-portfolio-standard>
    
and then later tag the technology with this constraint.
        <supplysector name="ethanol">
            <subsector name="corn ethanol">
               <technology name="corn ethanol">
                   <period year="2005">
                       <input-subsidy name="Corn-Ethanol-Floor"/>
                   </period>
                   <period year="2010">
                       <input-subsidy name="Corn-Ethanol-Floor"/>
                   </period>
                   <period year="2015">
                       <input-subsidy name="Corn-Ethanol-Floor"/>
                   </period>
                   <period year="2020">
                       <input-subsidy name="Corn-Ethanol-Floor"/>
                   </period>
 etc.

Reply:
Oops the second part isn't showing up, I'll try in a new comment.
        <supplysector name="ethanol">
            <subsector name="corn ethanol">
               <technology name="corn ethanol">
                   <period year="2005">
                       <input-subsidy name="Corn-Ethanol-Floor"/>
                   </period>
                   <period year="2010">
                       <input-subsidy name="Corn-Ethanol-Floor"/>
                   </period>
                   <period year="2015">
                       <input-subsidy name="Corn-Ethanol-Floor"/>
                   </period>
                   <period year="2020">
                       <input-subsidy name="Corn-Ethanol-Floor"/>
                   </period>
 etc.

Reply:
oh, that is a copy-paste error. Sorry, I will fix. There should not be a fixed-tax or an isFixedTax in that example.

Reply:
did i mention that i don;t like the github interface?  Or maybe I am not worthy.